advanced/Scripts/piholeDebug.sh

```diff
@@ -1076,18 +1076,27 @@ show_db_entries() {
     IFS="$OLD_IFS"
 }
 
+show_groups() {
+    show_db_entries "Groups" "SELECT * FROM \"group\"" "4 100 7 10 13 50"
```

Comment:
The column widths should be adjusted as there are only four columns in the `group` table.

Reply:
This item is actually still unresolved.